gtav/pillbox_hospital.lua

```diff
@@ -4,9 +4,30 @@ exports('GetPillboxHospitalObject', function()
 end)
 
 PillboxHospital = {
-    ipl = "rc12b_default",
+    default = 'rc12b_default',
+    fixed = 'rc12b_fixed',
+    destroyed = 'rc12b_destroyed', -- Destroyed and hospital interior need to be loaded at the same time
+    hospitalinterior = 'rc12b_hospitalinterior',
 
-    Enable = function(state)
-        EnableIpl(PillboxHospital.ipl, state)
-    end
+    Enable = function(ipl, state)
+        EnableIpl(ipl, state)
+    end,
+
+    Clear = function()
+        EnableIpl(PillboxHospital.default, false)
+        EnableIpl(PillboxHospital.fixed, false)
+        EnableIpl(PillboxHospital.destroyed, false)
+        EnableIpl(PillboxHospital.hospitalinterior, false)
+    end,
+
+    LoadDefault = function()
+        PillboxHospital.Clear()
+        -- [DEFAULT HOSPITAL]
+        --EnableIpl(PillboxHospital.default, true)
+        -- [FIXED HOSPITAL]
+        --EnableIpl(PillboxHospital.fixed, true)
+        -- [DESTROYED HOSPITAL]
+        EnableIpl(PillboxHospital.destroyed, true)
+        EnableIpl(PillboxHospital.hospitalinterior, true)
+    end,
```

Comment:
Please use your .Enable methods here, rather than calling "internal" EnableIIpl, also still breaking API/not quite consistent with other files.